src/commands/bundle.ts

```diff
@@ -0,0 +1,44 @@
+import log from "loglevel";
+import { sdkGeneratorApiHandler } from "../generateSdk/generateSdkApi.js";
+import { GenezioBundleOptions } from "../models/commandOptions.js";
+import { SdkGeneratorResponse } from "../models/sdkGeneratorResponse.js";
+import { getProjectConfiguration } from "../utils/configuration.js";
+import { ProjectConfiguration } from "../models/projectConfiguration.js";
+import { bundle } from "../bundlers/utils.js";
+import { mkdirSync } from "fs";
+import { writeToFile, zipDirectory } from "../utils/file.js";
+import path from "path";
+
+export async function bundleCommand(options: GenezioBundleOptions) {
```

Comment:
Same for this code. It's duplicated twice in `deploy.ts` line 258 and `generateSdk.ts` line 61. It would be nice to have 1 function instead of 3.